modules/Bio/EnsEMBL/Production/Pipeline/Xrefs/RNAcentralMapping.pm

```diff
@@ -31,14 +32,24 @@ sub run {
   my $xref_url     = $self->param_required('xref_url');
   my $species      = $self->param_required('species');
   my $base_path    = $self->param_required('base_path');
-  my $source_url   = $self->param_required('source_url');
+  my $db_url       = $self->param_required('source_url');
```

Comment:
This is confusing. Why have the local variable named differently to the hive variable?